tests/integration/bookmark.test.js

```diff
@@ -47,7 +47,6 @@ describe('POST /api/v1/:articleId/bookmark', () => {
     agent.get(`/api/v1/article/${invalidArticleId}/bookmark`)
       .set('Authorization', JWT_TOKEN)
       .end((_err, res) => {
-        assert.equal(res.status, 409);
```

Comment:
Nice work testing your endpoint. Is there any reason why you are not taking this line out? It would be better if you make assertions on the response status.

Reply:
it is outdated, I have already added the line back to the test